about.html

```diff
@@ -44,19 +37,19 @@
     </div>
   </header>
 <section class="about">
-  <h1 class="aboutTitle">BIGGEST MUSIC<br>FESTIVAL 2022</h1>
+  <h1 class="about-title">BIGGEST MUSIC<br>FESTIVAL 2022</h1>
   <p class="abou-textBox">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed molestie ante, a semper magna. Nam quis iaculis libero. Sed vehicula, augue et dapibus ornare, magna risus aliquet ipsum, in imperdiet ex arcu ac risus. Cras porta aliquam porttitor. Nunc sed tortor ipsum. Cras suscipit ex risus, sed consequat quam aliquet id. Cras hendrerit suscipit odio, sit amet venenatis eros vestibulum consectetur. Integer at enim sagittis, lacinia massa nec, condimentum ex.</p>
   <p class="about-normal-text">Please contact us through Email for any further question about the biggest music festival !</p>
   <a id="about-link" href="#">musicfestival@gmail.com <img src="https://img.icons8.com/external-those-icons-lineal-those-icons/10/000000/external-right-arrows-those-icons-lineal-those-icons-1.png" alt="close"/></a>
 </section>
 <section class="logo-section">
-  <h2 class="singerTitle">Music festival of the year 2022 </h2>
+  <h2 class="singertitle">Music festival of the year 2022 </h2>
   <div class="red-line"></div>
   <p class="about-normal-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent auctor dui eget purus interdum egestas. Aliquam quis dolor pharetra, blandit turpis non, rhoncus mauris. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Ut lobortis arcu vel porttitor maximus. Vestibulum non tellus vel ligula finibus posuere a a felis.</p>
   <div class="logo-border"><img id="ml-logo" src="images/logo-large.png" alt="courseLogo"></div>
 </section>
```

Comment:
- [ ] Your logo image in your about page does not display. It would be nice if you could check the image path and the image name so as to correct this issue.
- Your design:
![Capture1](https://user-images.githubusercontent.com/85113567/153715063-b8f23246-6b93-4d88-a0e5-7cb9d12b5011.PNG)

Reply:
This issue has been resolved